pkg/chunk/chunk_test.go

```diff
@@ -139,10 +139,10 @@ func TestChunksToMatrix(t *testing.T) {
 		"bar":  "baz",
 		"toms": "code",
 	}
-	chunk1 := dummyChunkFor(metric)
+	chunk1 := dummyChunkFor(model.Now(), metric)
```

Comment:
even more deterministic to use the same value of `now` here as at the beginning of the function?